README.md

```diff
@@ -41,51 +52,56 @@ CKEDITOR.plugins.addExternal('uploadcare', '/absolute/path/to/uploadcare/plugin.
 
 You can find more information about how to manually install plugins on [CKEditor Documentation][ck-docs-manual-install].
 
-## Configure
+## Usage
 
-Initialize a CKEditor plugin with additional params:
+Add `uploadcare` to the list of CKEditor plugins and toolbar.
+**Set your [public key][widget-docs-options-public-key]**. 
 
-```javascript
-UPLOADCARE_PUBLIC_KEY = "demopublickey"; // set public key for Uploadcare
-UPLOADCARE_LOCALE = 'ru'; // set locale if you wish
-CKEDITOR.replace( 'editor1', {
-  extraPlugins: 'uploadcare', // this will enable plugin
-  toolbar: [
-    // add Uploadcare button to toolbar, e.g.:
-    ['Bold', 'Italic', '-', 'NumberedList', 'BulletedList', '-', 'Link', 'Unlink', '-', 'Uploadcare']
-  ]
-});
+```
+CKEDITOR.replace('editor', {
+  extraPlugins: 'uploadcare',
+  toolbar: [['Uploadcare', /* Your toolbar items */]],
+  uploadcare: {publicKey: 'YOUR_PUBLIC_KEY'},
+})
 ```
 
-You can heavily customize widget behavior, i.e. file sources, file validation and much more, please
-read Uploadcare [widget][5] and [javascript API][6] documentation.
-
-## File Autostore
+## Configuration
 
-"Automatic file storing" **should** be enabled in your project settings.
-Please follow https://uploadcare.com/dashboard/ to ensure.
+Initialize a plugin with additional options:
 
-## Usage
+```javascript
+UPLOADCARE_LOCALE = 'ru' // set locale if you wish
+
+CKEDITOR.replace('editor', {
+  extraPlugins: 'uploadcare',
+  toolbar: [['Uploadcare', /* Your toolbar items */]],
+  uploadcare: {
+    publicKey: 'YOUR_PUBLIC_KEY',
+    multiple: true,
+    crop: '1:1,4:3',
+  },
+})
+```
 
-1. Press "Uploadcare" button.
-2. Select a file to upload and press "Upload"
-3. Wait for file to be uploaded.
-4. Crop an image as you wish.
-5. Click "Done". A cropped image will be available inside editor.
+You can heavily customize widget behavior, i.e. file sources, file validation and much more, please
+read [Uploadcare Widget][widget-docs-config] and [JavaScript API][widget-docs-js-api] documentation.
```

Comment:
You can deeply customize the widget behavior: file sources, file validation, and much more. Please, check out the Uploadcare Widget and JavaScript API documentation.